app/models/plant_trial.rb

```diff
@@ -8,6 +8,37 @@ class PlantTrial < ActiveRecord::Base
   include Filterable
   include Pluckable
 
+  validates :plant_trial_name,
+            presence: true
+
+  validates :project_descriptor,
+            presence: true
+
+  validates :plant_trial_description,
+            presence: true
+
+  validates :trial_year,
+            presence: true,
+            length: { is: 4 }
```

Comment:
All existing values are valid; I'm leaving this validation. I intend to submit another PR which converts datetime-like values to proper datetime types in the DB.